sdk/core/azure-core/README.md

```diff
@@ -117,7 +117,8 @@ int main()
 
 Note, the listener callback is executed on the same thread as the operation that triggered the log message. 
  It is recommended implementation due the minimal amount of log message processing on the callback thread.
-Where message processing is required, consider implementing in a way that the callback pushes the message string into a thread-safe queue, so that another thread would pick the messages from that queue and handle them.
+Where message processing is required, consider implementing in a way that the callback pushes the message 
+string into a thread-safe queue, so that another thread would pick the messages from that queue and handle them.
```

Comment:
This is only a line break change and is purely cosmetic. Unlike source code, such changes are unnecessary in markdown, and in fact make it harder to keep up consistently in different IDEs. We should avoid such line breaks and unrelated changes in PRs.